src/lib.rs

```diff
@@ -5,7 +5,7 @@
 //! [rust-vmm](https://github.com/rust-vmm/community) components.
 
 #![deny(missing_docs)]
-#[cfg(unix)]
+#[cfg(linux)]
 #[macro_use]
```

Comment:
Does bitflags only work on Linux?

Reply:
It's only used by the Linux-specific code, so when you don't compile in the 'unix' directory, bitflags is unused (so Rust complains about an unused dependency).